README.md

```diff
@@ -1003,6 +1023,12 @@ Updating topic
 {"topic" : {"name" : "t", "partitions" : {"0" : "0, 1"}, "options" : {"flush.ms" : "1000"}}}
 ```
 
+Deleting topic
+```
+# curl "http://localhost:7000/api/topic/delete?topic=t"
+{"topic" : {"name" : "t", "partitions" : {"0" : "0, 1"}, "options" : {}}}
+```
+
```

Comment:
@liorze I agree, however I made this consistent with the rest of the code (remove broker) which uses GET for the removal operation as well. Changing to proper HTTP DELETE operations it's, hence a change of the current API which is outside of the scope of this issue IMO.